ci-operator/step-registry/cucushift/installer/check/user-provisioned-dns/cucushift-installer-check-user-provisioned-dns-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: cucushift-installer-check-user-provisioned-dns
```

Comment:
As the cluster can not be created successfully, this step can not be verified now, we can fix it later if there are any issues once the cluster is created.

Reply:
I think this is a good case of test-driven development, the test could be run as soon as the code gets merged, and as development goes on, we can keep filling and fixing the post-check step.